cmd/gobrew/main.go

```diff
@@ -145,22 +176,21 @@ func isArgAllowed() bool {
 
 // Find takes a slice and looks for an element in it. If found it will
 // return it's key, otherwise it will return -1 and a bool of false.
-func Find(slice []string, val string) (int, bool) {
-	for i, item := range slice {
+func Find(slice []string, val string) bool {
+	r := regexp.MustCompile("([0-9]+).([0-9]+)*|beta.*|rc.*|latest")
+	if matches := r.FindString(val); matches != "" {
+		return true
+	}
+
+	for _, item := range slice {
 		if item == val {
-			return i, true
+			return true
 		}
 	}
-	return -1, false
+	return false
 }
 
-func usage() string {
-	usageMsg :=
```

Comment:
If you keep this var declaration it'd keep the development experience simple.

```
╰─$ go run cmd/gobrew/main.go -h
# command-line-arguments
cmd/gobrew/main.go:231:5: undefined: usageMsg
```

Reply:
because this constant is defined in other files. And everything works fine if you specify the package:
```sh
go run ./cmd/gobrew -h

gobrew dev

Usage:

    gobrew use <version>           Install and set <version>
    gobrew ls                      Alias for list
    gobrew ls-remote               List remote versions (including rc|beta versions)
```